modules/extensions/autoscaler.tf

```diff
@@ -20,6 +20,7 @@ locals {
   cluster_autoscaler_manifest_path = join("/", [local.yaml_manifest_path, "cluster_autoscaler.yaml"])
   cluster_autoscaler_defaults = {
     "cloudProvider"                              = "oci-oke",
+    "extraArgs.expander"                         = "priority",
```

Comment:
Can we make this a variable instead, that we can pass through from where we call the module?  Since for the e2e cluster (for example) I don't think we'll want preemptible so we won't need this right

Reply:
Make sense, let me see how to do that.